eisbuk-admin/src/utils/helpers.ts

```diff
@@ -103,3 +103,23 @@ can be published at https://appname--new-feature-randomhash.web.app/
 */
 export const getOrgFromLocation = (location: string): string =>
   location.replace(/--[^.]+/, "");
+
+export const getOrganization = (): string => {
+  return (
+    localStorage.getItem("organization") ||
+    getOrgFromLocation(window.location.hostname)
+  );
+};
```

Comment:
I think that the `helpers` should be some function which process a given value (pure functions), not read from store, local store or somewhere else...for that I think that you can create a new file: `@/lib/getters.ts` and for now just put this function, but we can use it in the future for any other getters (if there will be any)

Reply:
Also, here you can use `__organization__` as a fallback (as it handles additional case where the window is not defined)